docker/pypi/wmagent/Dockerfile

```diff
@@ -12,10 +12,11 @@ ENV LD_LIBRARY_PATH=/usr/lib/oracle
 ENV PATH=$PATH:/usr/lib/oracle
 ENV PKG_CONFIG_PATH=/usr/lib/oracle
 
+# This TAG is overwritten by WMCore GH actions workflow
+ENV TAG=X.Y.Z
```

Comment:
Hi @amaltaro, 

Even though I can see this approach may work here, upon finding out how those images are built (from this discussion [here](https://github.com/dmwm/WMCore/issues/8797#issuecomment-1618907933)), I now really object adding yet another workaround just to keep running with a sub optimal CI/CD pipeline. 

I have struggled to find an elegant and natively supported by Docker solution for providing the `WMA_TAG` as an external option to the Dockerfile and to the build process. The continuation of using `sed` based hacks [1] to me is a drawback and future obstacle in many ways. I'd be more in favor of finding the way to fully utilize the Docker power in our CI pipeline. Like: 
* Adding the possibility to pass parameters to the docker build process 
* Downloading the full part of the project area required for a given build not only the `Dockefile` so we do not deprive ourselves of using basic functionalities of the containerization system, like `ADD` commands etc.  

[1]
https://github.com/dmwm/WMCore/blob/762bae943528241f67625016fd019ebcd0014af1/.github/workflows/docker_images_template.yaml#L29